src/app/routes/explorer/Milestone.tsx

```diff
@@ -141,6 +142,19 @@ class Milestone extends AsyncComponent<RouteComponentProps<MilestoneRouteProps>,
         const result = await this._tangleService.milestoneDetails(Number.parseInt(index, 10));
 
         if (result) {
+            try {
+                const tangleService = ServiceFactory.get<TangleService>("tangle");
+                const info = await tangleService.info();
+
+                this.setState({
+                    messageId: messageIdFromMilestonePayload(info.protocol.protocolVersion, result)
+                });
+            } catch (error) {
+                if (error instanceof Error) {
+                    console.log(error.message);
```

Comment:
Do we need to display some notification here or are we deliberately just logging it?